commands/Moderation/kick.js

```diff
@@ -55,8 +56,8 @@ class Kick extends Command {
             data.guild.save();
 
             let caseInfo = {
-                channel: message.channel,
-                moderator: message.author,
+                channel: message.channel.name,
```

Comment:
It's better to save the ID of the channel (`message.channel.id`) instead of the name